faqs/galaxy/datasets_moving_datasets_between_galaxy_servers.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Moving datasets between Galaxy servers
+area: datasets
+box_type: tip
+layout: faq
+contributors: [jennaj, Melkeb]
+---
+
+For individual datasets:
+1. Click on name of file (dataset) to expand the info.
```

Comment:
would filename work better here?

Reply:
also, before this first item, we could say something like:

`In the origin server:`

Reply:
I changed it to name of dataset to make it more clearer and added the first line about the origin server similar as the destination server text.

Reply:
ok, thanks! that makes sense